storybook/storybook-react/src/components/grid/PageGrid.stories.tsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { GridCell, PageGrid, Paragraph } from '@amsterdam/design-system-react/src'
+import { Meta, StoryObj } from '@storybook/react'
+
+import '@amsterdam/design-system-css/src/grid/page-grid.scss'
+import '@amsterdam/design-system-css/src/grid/grid-cell.scss'
+import '@amsterdam/design-system-css/src/heading/heading.scss'
+import '@amsterdam/design-system-css/src/paragraph/paragraph.scss'
+import '../../../../storybook-css/src/docs/stories.scss'
+
+const meta = {
+  title: 'PageGrid',
+  id: 'page-grid',
+  component: PageGrid,
+} satisfies Meta<typeof PageGrid>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  args: {
+    children: (
+      <>
+        {Array.from(Array(12).keys()).map((i) => (
+          <GridCell key={i}>
+            <div className="amsterdam-docs-pink-box">
+              <Paragraph>{i + 1}</Paragraph>
+            </div>
+          </GridCell>
+        ))}
+      </>
+    ),
+  },
+  parameters: {
+    docs: {
+      layout: 'centered',
+    },
+  },
+}
+
+export const ThreeBlocks: Story = {
+  args: {
+    children: (
+      <>
+        <GridCell gridColumns={[1, 4]}>
```

Comment:
This is a compact API. Is it readable enough? Can people infer what it means and get used to it?

The obvious alternative is `<GridCell start={1} end={4}>`. Probably even better. But to allow all components to be placed on a grid, these prop names are too generic. I’d say we’d prefer something like `<Component grid-column-start={1} grid-column-end={4}>` or `<Component grid-column={[1, 4]}>`.

Additionally, we must consider the need to specify grid placement for all breakpoints, of which we currently have two, e.g. `<Component grid-column={{narrow: [1, 1], wide: [1, 4]}>`. Is this acceptable?